src/main/java/org/folio/inventory/domain/instances/titles/PrecedingSucceedingTitle.java

```diff
@@ -44,4 +44,30 @@ public static PrecedingSucceedingTitle from(JsonObject rel, String title, String
       rel.getString(SUCCEEDING_INSTANCE_ID_KEY),
       title, hrid, identifiers);
   }
+
+  public JsonObject toPrecedingTitleJson() {
+    return toJson(PRECEDING_INSTANCE_ID_KEY, precedingInstanceId);
+  }
+
+  public JsonObject toSucceedingTitleJson() {
+    return toJson(SUCCEEDING_INSTANCE_ID_KEY, succeedingInstanceId);
+  }
+
+  private JsonObject toJson(String succeedingInstanceIdKey, String succeedingInstanceId) {
```

Comment:
It appears these parameters are not always related to the succeeding instance, could they be renamed 

```suggestion
  private JsonObject toJson(String instanceIdKey, String instanceId) {
```

Reply:
done